src/os/bluestore/BlueFS.cc

```diff
@@ -3040,18 +3255,55 @@ void BlueFS::wait_for_aio(FileWriter *h)
 }
 #endif
 
-int BlueFS::_flush(FileWriter *h, bool force, std::unique_lock<ceph::mutex>& l)
+void BlueFS::append_try_flush(FileWriter *h, const char* buf, size_t len)
```

Comment:
Can't find any references to this func. Remove?

Reply:
BlueRocksEnv uses it for `BlueRocksWritableFile::Append()`.

Reply:
yep. missed that